index.js

```diff
@@ -1,18 +1,222 @@
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
-}
+const readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+// Menu for user to select game
+async function chooseGame() {
+  gameChoice = await ask(
+    ` Welcome! \n Which game would you like to play? \n [1.] Computer guesses your number \n [2.] You guess the computer's number \n [X] Press X key to exit. ${prompt}`
+  );
+  while (
+    gameChoice.toLowerCase() !== "x" &&
+    gameChoice.toLowerCase() !== "2" &&
+    gameChoice.toLowerCase() !== "1"
+  ) {
+    gameChoice = await ask(
+      ` \n Oops! Let's try this again \n Which game would you like to play? \n [1.] Computer guesses your number \n [2.] You guess the computer's number \n [X] Press X key to exit. ${prompt}`
+    );
+  }
+}
+
+//This is the First game where the computer guesses the user's number.
+async function game1() {
+  await rangeUser(); // user sets the range
+  //console.log(rangeMax); // debug check
+  let random = randomNum(rangeMin, rangeMax); //assigns the random guess to a variable
+  let guess = random;
+  let answer = "a";
+  numTries += 1;
+
+  while (correct === false) {
+    //console.log(numTries);
+    //answer = await ask(` Is your number ${guess}? \n \[Y\] or \[N\] ${prompt}`);
+    //numTries += 1;
+    while (
+      answer[0].toLowerCase !== "h" ||
+      answer[0].toLowerCase !== "l" ||
+      answer[0].toLowerCase !== "y"
+    ) {
+      answer = await ask(
+        `My guess is ${guess}. \n Is this correct? \[Y\] \n or is your number \[H\]igher or \[L\]ower? ${prompt}`
+      );
+      console.log(numTries);
+
+      // THIS CHEATER CHECK DOES NOT WORK YET
+
+      // if (answer[0].toLowerCase === 'h' && guess > rangeMax) {
+      //   console.log(
+      //     ` Your answer of ${answer} is out of the range specified. \n We don't tolerate cheaters up in these parts! `
+      //   );
+      //   correct === true;
+      //   process.exit();
+      // } else
+
+      if (answer[0].toLowerCase() === "y") {
+        correct = true;
+        console.log(`Your number was ${guess}!`);
+        console.log(`This took ${numTries} tries.`);
+        correct === true;
+        break;
+      } else if (answer[0].toLowerCase() === "h") {
+        rangeMin = guess + 1;
+        guess = Math.floor((rangeMax - rangeMin) / 2 + rangeMin);

```

Comment:
Use this as your first guess as well to finish the make it smarter story... it's behind by one guess currently